### ISSUE TITLE
Blur region for GTK context menus is too large

### ISSUE THREAD

**Issue by taj-ny** [taj-ny/kwin-forceblur#17]
![image](https://github.com/taj-ny/kwin-forceblur/assets/79316397/a8997479-ed39-48ff-b7a4-346354cd40a3)

**Comment by D3SOX** [comment]
Seems to depend on some settings as it does not happen to me

![image](https://github.com/taj-ny/kwin-forceblur/assets/24937357/b38e23b1-e372-431c-a4fe-2af51063adee)

Are you running Firefox with Wayland or XWayland?

**Comment by taj-ny** [comment]
Wayland. I use a transparent Firefox theme so I have it blurred.

**Comment by D3SOX** [comment]
Yeah, I was posting this too fast. Maybe I need to restart Firefox for it to happen (don't want to do that right now)

But I could reproduce it Chromium now

![image](https://github.com/taj-ny/kwin-forceblur/assets/24937357/5b27fe0f-e280-4f78-a6bc-a2c8ad3b7096)

**Comment by taj-ny** [comment]
It may be possible to get the size of the borders from ``_GTK_FRAME_EXTENTS`` and then shrink the blur region, but that'll only work on X11. On Wayland there's nothing I can do aside from adding the ability to shrink the region manually.
